src/Uno.UI/UI/Xaml/Controls/Primitives/LayoutInformation.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Windows.Foundation;
+
+namespace Windows.UI.Xaml.Controls.Primitives
+{
+	partial class LayoutInformation
+	{
+		public static Rect GetLayoutSlot(FrameworkElement element)
```

Comment:
Nice method to have! UWP has a funny habit of hiding useful layout information in static helper classes

Reply:
Yep.